evm/src/main/java/org/hyperledger/besu/evm/worldstate/AbstractWorldUpdater.java

```diff
@@ -188,4 +209,28 @@ protected void reset() {
     updatedAccounts.clear();
     deletedAccounts.clear();
   }
+
+  @Override
+  public void addCodeToEOA(final Address address, final Bytes code) {
+    if (temporaryEOACode.containsKey(address)) {
+      return;
```

Comment:
what happens if there are multiple 7702 transactions in a block for a single address?  should we be doing a codeHash check here?  or a nonce check?